OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -337,10 +410,13 @@ public void SetListenerPosition(WPos position)
 		{
 			// Move the listener out of the plane so that sounds near the middle of the screen aren't too positional
 			AL10.alListener3f(AL10.AL_POSITION, position.X, position.Y, position.Z + 2133);
+			CheckALError();
 
 			var orientation = new[] { 0f, 0f, 1f, 0f, -1f, 0f };
 			AL10.alListenerfv(AL10.AL_ORIENTATION, orientation);
+			CheckALError();
 			AL10.alListenerf(EFX.AL_METERS_PER_UNIT, .01f);
+			CheckALError();
```

Comment:
This kills us on macOS because the native AL library doesn't support `EFX.AL_METERS_PER_UNIT`.

We then hit a second level of trouble with: `exception inside UnhandledException handler: Cannot access a disposed object.
`

Reply:
Dropped the error checking from the PR, since it is indeed orthogonal to the goal here and is hitting latent issues.